src/aurelia-slickgrid/custom-elements/aurelia-slickgrid.ts

```diff
@@ -544,6 +546,7 @@ export class AureliaSlickgridCustomElement {
 
     // dispose the Components
```

Comment:
actually missing the dispose for the empty warning
```ts
this.slickEmptyWarning?.dispose();
```

Reply:
seem to be missing aurelia-slickgrid unit test changes as well on the footer/pagination side of things